examples/training/classification/imagenet/basic_training.py

```diff
@@ -85,21 +85,41 @@
     "Keyword argument dictionary to pass to the constructor of the model being trained",
 )
 
+flags.DEFINE_string(
+    "learning_rate_schedule",
+    "ReduceOnPlateau",
+    "String denoting the type of learning rate schedule to be used",
+)
+
+flags.DEFINE_string(
```

Comment:
For this and the percentage below (and epochs), should be DEFINE_integer

Reply:
Done

Reply:
Done